### ISSUE TITLE
Add TRES info to emails

### ISSUE THREAD

**Issue by neilmunday** [neilmunday/slurm-mail#142]
Hi! 
Is it possible to add GPU number to the templates?
In scontrol output it appers in *TRES fields:

   NumNodes=1 NumCPUs=6 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   ReqTRES=cpu=5,mem=90000M,node=1,billing=5,gres/**gpu=1** 
   AllocTRES=cpu=6,mem=90000M,node=1,billing=6,gres/**gpu=1**

_Originally posted by @aav18141 in https://github.com/neilmunday/slurm-mail/discussions/14#discussioncomment-10846176_

**Comment by neilmunday** [comment]
Looks like I should be able to get the TRES information from `sacct` as well.

**Comment by neilmunday** [comment]
@aav18141 are you table to show me the output from the `sacct` command for a job that used some GPUs, e.g.

```
sacct --fields JobID,AllocTRES -P
```

**Comment by dstam** [comment]
Would indeed be interesting to add TRES aswell to the mails. I have pasted the output of two jobs from our environment:

```
JobID|AllocTRES
8117657|billing=64,cpu=9,gres/cpu=9,gres/gpu:a100_3g.20gb=1,gres/gpu=1,mem=60G,node=1
8117657.batch|cpu=9,gres/cpu=9,gres/gpu:a100_3g.20gb=1,gres/gpu=1,mem=60G,node=1
8117657.extern|billing=64,cpu=9,gres/cpu=9,gres/gpu:a100_3g.20gb=1,gres/gpu=1,mem=60G,node=1
8117678|billing=192,cpu=16,gres/cpu=16,gres/gpu:h100=1,gres/gpu=1,mem=180G,node=1
8117678.batch|cpu=16,gres/cpu=16,gres/gpu:h100=1,gres/gpu=1,mem=180G,node=1
8117678.extern|billing=192,cpu=16,gres/cpu=16,gres/gpu:h100=1,gres/gpu=1,mem=180G,node=1
```

Thanks in advanced for adding it

**Comment by neilmunday** [comment]
@dstam @aav18141 would you prefer to see the entire TRES string in the Slurm-Mail e-mails or just GPU info?

**Comment by dstam** [comment]
Hi @neilmunday,

Would like to see all information of the TRES, as not all our users use GPU's

**Comment by neilmunday** [comment]
Thanks for the feedback @dstam.

I've created a mod locally that adds a TRES table to e-mails for jobs that have ended:

<img width="360" alt="Slurm-Mail 4 22 TRES" src="https://github.com/user-attachments/assets/d443697d-e9ff-4bc6-b4e4-2666878253a7">

What do you think?

**Comment by dstam** [comment]
@neilmunday Looks nice

**Comment by neilmunday** [comment]
Feature implemented successfully in 4.22 branch.

**Comment by neilmunday** [comment]
Version 4.22 released
